concepticondata/references/references.bib

```diff
@@ -3437,4 +3437,47 @@ @Article{Rowe1954b
   pages                               = {137-146}
 }
   
-  
+@Article {Johansson2020,
+  Author                              = {Johansson, Niklas Erben and Anikin, Andrey and Carling, Gerd and Holmer, Arthur},
+  Title                               = {The typology of sound symbolism: Defining macro-concepts via their semantic and phonetic features},
+  Journal                             = {Linguistic Typology},
+  Year                                = {2020},
+  Publisher                           = {De Gruyter Mouton},
+  Address                             = {Berlin, Boston},
+  Volume                              = {24},
+  Number                              = {2},
+  Doi                                 = {https://doi.org/10.1515/lingty-2020-2034},
```

Comment:
leave out the `https://doi.org/` it is redundant (and not used below).

Reply:
Ah, and the capitalization is not consistent (I know it is not in our whole bibliography, but we tend to not capitalize titles or booktitles) (refers to the entry below).